target_eventbridge/helpers/aws/ebHelper.py

```diff
@@ -0,0 +1,42 @@
+import boto3
+from botocore.config import Config
+import datetime
+import json
+import os
+
+from target_eventbridge.helpers.constantsHelper import (
+    AWS_REGION,
+)
+
+def sendEvent(event_bus_name, event_detail_type, event_source, tapData):
+
+    if tapData == {}:
```

Comment:
More of a question, is it possible for tapData to be "empty" but not be an object? Not sure if we need to capture other error states. Can also be an enhancement down the road.